### DISCUSSION TITLE
UWB in TWR mode, locating packet conflicts

### DISCUSSION THREAD

**Discussion by lalalalang999** [bitcraze/discussions#786] [Q&A]
In UWB TWR mode, 4 ANCHOR IDs of 0-3 were used and 1 TAG was used for ranging and positioning normally. However, when using two TAGs, the TAG will frequently get stuck (suddenly 1s does not update the range data), please ask what setting is needed to use 2 or more TAGs in TWR mode.

**Comment by lalalalang999** [comment]
In TWR mode, does the UWB need to set other settings, such as setting the ID number of the TAG, when using multiple TAGs? In the case of 4 ANCHORs and 2 TAGs, when one TAG is ranging with ANCHOR 0, will the other TAG also be ranging with ANCHOR, and will this conflict with each other and cause one of the TAGs and ANCHOR to fail in ranging? Is this a random conflict if multiple TAGs are used?

**Comment by krichardsson** [comment]
TWR unfortunately only supports one tag. You are completely right, the two tags will range with the same anchor, and it is likely that packets will collide.

The TD0A modes are intended for multiple tags though and this is probably what you want to use,
